Casks/nozbe.rb

```diff
@@ -0,0 +1,25 @@
+cask "nozbe" do
+  version "2023.14"
+  sha256 "92cd7aa7855567ac3a563c481caf1bf0f795cd9efc96f20516ea77186448c990"
+
+  url "https://builds.nozbe.com/mac/#{version}/Nozbe.app.zip"
+  name "Nozbe"
+  desc "Project management app"
+  homepage "https://nozbe.com/"
+
+  livecheck do
+    url "https://nozbe.help/general/release-notes"
+    regex(%r{newest["']>version\s*?(\d+(?:\.\d+)*).*?</}i)
```

Comment:
```suggestion
    regex(%r{id=["']newest["'][^>]*?>\s*version\s*(\d+(?:\.\d+)+)}i)
```

* We're specifically targeting the `#newest` element, so there's something to be said for being explicit about the `id` attribute (i.e., we don't want to unexpectedly match an element with a different attribute value that happens to end with `newest`).
* There may be additional attributes on this element in the future, so we should account for those (i.e., `[^>]*?>`) to help keep the regex from breaking.
* In this particular case, we don't really benefit from anchoring the end using `.*?</`, so it's probably fine to leave it off for now.